### ISSUE TITLE
Fix TCP protocol is not supported when calling gradle remote api and DOCKER_HOST is set

### ISSUE THREAD

**Issue by toplac** [toplac/cloud-native-zwitscher#1]
**Description**

**Steps to reproduce the issue:**
1. Clone repository and change into folder
1. `./gradlew dockerVersion`

**Results:** 

```
:dockerVersion
Retrieving Docker version.
[Daemon worker Thread 3] INFO org.apache.http.impl.execchain.RetryExec - I/O exception      (org.apache.http.conn.UnsupportedSchemeException) caught when processing request: tcp protocol is  not supported
[Daemon worker Thread 3] INFO org.apache.http.impl.execchain.RetryExec - Retrying request
[Daemon worker Thread 3] INFO org.apache.http.impl.execchain.RetryExec - I/O exception (org.apache.http.conn.UnsupportedSchemeException) caught when processing request: tcp protocol is not supported
[Daemon worker Thread 3] INFO org.apache.http.impl.execchain.RetryExec - Retrying request
[Daemon worker Thread 3] INFO org.apache.http.impl.execchain.RetryExec - I/O exception (org.apache.http.conn.UnsupportedSchemeException) caught when processing request: tcp protocol is not supported
[Daemon worker Thread 3] INFO org.apache.http.impl.execchain.RetryExec - Retrying request
:dockerVersion FAILED

FAILURE: Build failed with an exception.

* What went wrong:
Execution failed for task ':dockerVersion'.
> org.apache.http.conn.UnsupportedSchemeException: tcp protocol is not supported
```

**Expected Results:**

Output similiar to the one when calling docker version

```
Client:
Version:      1.12.0
 API version:  1.24
 Go version:   go1.6.3
 Git commit:   8eab29e
 Built:        Thu Jul 28 23:54:00 2016
 OS/Arch:      windows/amd64

Server:
Version:      1.12.1
 API version:  1.24
 Go version:   go1.6.3
 Git commit:   23cf638
 Built:        Thu Aug 18 17:52:38 2016
 OS/Arch:      linux/amd64
```

**Additional Information:**

The following Environment Variables are set:

```
$ docker-machine.exe env
export DOCKER_TLS_VERIFY="1" 
export DOCKER_HOST="tcp://192.168.99.100:2376"
export DOCKER_CERT_PATH="C:\Users\t.placht\.docker\machine\machines\default"
export DOCKER_MACHINE_NAME="default"
```

The following works: 

```
DOCKER_HOST=https://192.168.99.100:2376 ./gradlew dockerVersion
```

**Comment by ghost** [comment]
Your issue doesn't follow our guidelines. Please fix the following:
- Issue must have at least one label [(?)](https://gitmagic.io/rules/#/issue/label-must-be-set)
- Issue title must be in imperative, present tense (e.g. "add", "fix", "change") [(?)](https://gitmagic.io/rules/#/issue/subject-must-be-in-tense)

[Click here](https://app.gitmagic.io/toplac/cloud-native-zwitscher/1/b7db07e5-7715-4681-b612-918edb969205/37d73e08-c780-4fbb-804e-ccfe0bd2351d) for details.

Thank you! :pray:
<sub>
This comment was made by [GitMagic](https://gitmagic.io/) – Magically enforcing your contribution guidelines.
</sub>

**Comment by ghost** [comment]
This issue still violates some of our guidelines:
- Issue must have at least one label [(?)](https://gitmagic.io/rules/#/issue/label-must-be-set)

[Click here](https://app.gitmagic.io/toplac/cloud-native-zwitscher/1/b7db07e5-7715-4681-b612-918edb969205/37d73e08-c780-4fbb-804e-ccfe0bd2351d) for details.

**Comment by ghost** [comment]
Thank you, everything looks good now! :bowtie:

**Comment by toplac** [comment]
Fixed in upstream with 44c7ffb643b20fd85e114ce94445d98ac2530a1b
